lsp/nls/src/linearization/mod.rs

```diff
@@ -403,6 +404,7 @@ impl<'a> Linearizer for AnalysisHost<'a> {
                             final_term_pos(body)
                         }
                         Term::Op1(UnaryOp::StaticAccess(field), _) => &field.pos,
+                        Term::Fun(ident, _) | Term::FunPattern(Some(ident), ..) => &ident.pos,
```

Comment:
Are we coping with the bug by searching for the indent position, because it's mistakenly stored instead of the function body? No, please, let's not add a new strange behavior on top of a bug. We can do that in a subsequent PR but the right thing to do is to fix the behavior that registers the position of the function as the position of its argument.

Reply:
Okay